core-bundle/src/Resources/contao/templates/forms/form_range.html5

```diff
@@ -17,5 +17,5 @@
     <p class="error"><?= $this->getErrorAsString() ?></p>
   <?php endif; ?>
 
-  <input type="<?= $this->type ?>" name="<?= $this->name ?>" id="ctrl_<?= $this->id ?>" class="range<?php if ($this->class): ?> <?= $this->class ?><?php endif; ?>"<?php if ($this->value): ?> value="<?= (float) $this->value ?>"<?php endif; ?><?= $this->getAttributes() ?>>
+  <input type="<?= $this->type ?>" name="<?= $this->name ?>" id="ctrl_<?= $this->id ?>" class="range<?php if ($this->class): ?> <?= $this->class ?><?php endif; ?>"<?php if ('' !== $this->value && null !== $this->value): ?> value="<?= (float) $this->value ?>"<?php endif; ?><?= $this->getAttributes() ?>>
```

Comment:
```suggestion
  <input type="<?= $this->type ?>" name="<?= $this->name ?>" id="ctrl_<?= $this->id ?>" class="range<?php if ($this->class): ?> <?= $this->class ?><?php endif; ?>"<?php if ('' !== (string) $this->value): ?> value="<?= (float) $this->value ?>"<?php endif; ?><?= $this->getAttributes() ?>>
```

This is how we have solved it everywhere else. It will also handle `false`, which can never be a valid option.